cli/azd/resources/scaffold/templates/ai-models.bicept

```diff
@@ -0,0 +1,198 @@
+{{define "ai-models.bicep" -}}
+{{- if .AiFoundryProject }}
+{{- range .AiFoundryProject.Models }}
+param {{bicepName .Name}}{{bicepName .Version}}Location string
+{{- end }}
+{{- end }}
+
+@description('Tags that will be applied to all resources')
+param tags object = {}
+
+@description('Main location for the resources')
+param location string
+
+var abbrs = loadJsonContent('./abbreviations.json')
+var resourceToken = uniqueString(subscription().id, resourceGroup().id, location)
+
+@description('The name of the environment')
+param envName string
+
+module keyVault 'br/public:avm/res/key-vault/vault:0.6.1' = {
+  name: 'keyvaultForHub'
+  params: {
+    name: '${abbrs.keyVaultVaults}hub${resourceToken}'
+    location: location
+    tags: tags
+    enableRbacAuthorization: false
+  }
+}
+
+module storage 'br/public:avm/res/storage/storage-account:0.17.2' = {
+  name: 'storageAccountForHub'
+  params: {
+    tags: tags
+    name: '${abbrs.storageStorageAccounts}${resourceToken}'
```

Comment:
This collides with the azd storage account: `var storageAccountName = '${abbrs.storageStorageAccounts}${resourceToken}'` -- Do we need a hub-specific token?